.idea/misc.xml

```diff
@@ -1,5 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="DesignSurface">
```

Comment:
you needn't commit files from the .idea folder. please remove this one from the pull request

in order to prevent such files to get indexed, try to update the .gitignore file with 
```
*.iml
.idea/
```